lib/module.js

```diff
@@ -40,6 +40,197 @@ const preserveSymlinks = !!process.binding('config').preserveSymlinks;
 const experimentalModules = !!process.binding('config').experimentalModules;
 
 const errors = require('internal/errors');
+// Fails with no such built-in module: frenemies
+// TODO: why?  Do I need to add something to a Makefile or load order
+// list somewhere?
```

Comment:
Are you looking for `node.gyp`?

Reply:
Probably.  Thanks for pointing me at that.  Afk, but will try that soon.